driver-mct/main/cime_comp_mod.F90

```diff
@@ -4243,7 +4243,7 @@ subroutine cime_run_atmocn_setup(hashint)
        ! ocn budget
        !----------------------------------------------------------
        if (do_budgets) then
-          call cime_run_calc_budgets3()
+          call cime_run_calc_budgets3(in_cplrun=.true.)
```

Comment:
these cime_run_calc_budgetsX routines are each called once.  They don't need arguments.  Just change the value in the timer call in the routine.
What about cime_run_calc_budgets2?

Reply:
Seems dangerous to me. These are subroutines that could be called from multiple locations, or the context to the call could be changed. The comments for some of these routines also seem out of date (RASM2 only for cime_run_calc_budgets3?). Seemed more consistent to take this approach. These are optional parameters, so don't have to be used.

> What about cime_run_calc_budgets2?

I do not understand the question.

Reply:
does the call to cime_run_calc_budgets2 need similar modifications?

Reply:
No. It is not called within the scope of an existing CPL:RUN timer. 

cime_run_calc_budgets3 is called from cime_run_atmocn_setup, and it is within cime_run_atmocn_setup that the CPL:RUN timer is started before the call to cime_run_calc_budgets3.

cime_run_calc_budgets1 and cime_run_calc_budgets2 are both called from cime_run and are not already with an active CPL:RUN timer.

Reply:
These cime_run_calc_budget subroutines were created just to make the main integration loop have fewer LOC.  They won't be called twice but might be moved.